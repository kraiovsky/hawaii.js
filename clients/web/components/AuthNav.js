import React from 'react'
import Link from 'next/link'

export default () => {
  return (
    <>
      <Link href="/auth">
        <a>Login</a>
      </Link>
      {' | '}
      <Link href="/auth">
        <a>Signup</a>
      </Link>
    </>
  )
}
