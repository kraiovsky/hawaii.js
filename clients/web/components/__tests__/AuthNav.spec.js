import React from 'react'
import renderer from 'react-test-renderer'
import AuthNav from '../AuthNav'

test('Render AuthNav', () => {
  const component = renderer.create(<AuthNav />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
