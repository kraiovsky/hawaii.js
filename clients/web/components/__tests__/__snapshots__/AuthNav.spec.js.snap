// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render AuthNav 1`] = `
Array [
  <a
    href="/auth"
    onClick={[Function]}
  >
    Login
  </a>,
  " | ",
  <a
    href="/auth"
    onClick={[Function]}
  >
    Signup
  </a>,
]
`;
